### PR TITLE
Feature/limit owner permissions

### LINKED ISSUE #161: Most fields should not be editable on a "registered" rabbit
If a rabbit is registered and put in the "active" population and got a certificate number. The fields should only be editable by the Genebank owner and not by herd owner. 

Herd owner should be able to add a note. Add weights and hull. And be able to sell the rabbit to External herd or another internal hed

### PR DESCRIPTION
**Pull request by norling**
Makes sure that owners can only edit notes in the individual editing form.

Closes #161

### CODE REVIEW

File: frontend/src/individual_edit.tsx [hunk 1]
```diff
@@ -241,7 +248,7 @@ export function IndividualEdit({id}: {id: string | undefined}) {
                 onChange={(event) => {updateField('number', event.currentTarget.value)}}
               />
               <TextField
-                disabled={!user?.canEdit(individual.genebank)}
+                disabled={!canManage}
```
Comment (danr) [inline frontend/src/individual_edit.tsx]: No need to check `isNew` here?

**Review by danr — APPROVED** [review]
Looks good

File: frontend/src/individual_edit.tsx (hunk 1 above)
Comment (jhagberg) [reply frontend/src/individual_edit.tsx]: Yes a herd owner might want to input a new rabbit In the system that use a old system aka a paper certificate and if so they would like to input the certificate number from the physical paper I'm the system.
If the the herd owner want to issue a digital certificate of the New rabbit the certificate number should come from our system. So possible three different number series of the certificate number.

**Review by jhagberg — CHANGES_REQUESTED** [review]
Herd owner might want to input certificate numbers of they write a paper certificate for the rabbit and not issue a digital certificate from the our system.

File: frontend/src/individual_edit.tsx (hunk 1 above)
Comment (norling) [reply frontend/src/individual_edit.tsx]: ok! :) I had understood things as the certificate always coming from the manager, but that's an easy fix!

**Review by jhagberg — APPROVED** [review]
Good

**Pull request merged**